helm_deploy/values-dev.yaml

```diff
@@ -24,6 +24,8 @@ env:
   OAUTH_API_BASE_URL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
   WIND_BACK_SECONDS: "10"
   APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
+  APPLICATION_LISTENER_TOTALDELAYDURATION: "PT45M"
+  APPLICATION_LISTENER_DELAYDURATION: "PT10M"
 
```

Comment:
once tested in dev this will be reduced to 1 minute where as prod will remain at 45 minutes